pkg/ccl/crosscluster/logical/lww_row_processor_test.go

```diff
@@ -420,6 +418,136 @@ func TestLWWConflictResolution(t *testing.T) {
 				}
 				runner.CheckQueryResults(t, fmt.Sprintf("SELECT * from %s", tableNameDst), expectedRows)
 			})
+			t.Run("remote-update-with-outdated-previous", func(t *testing.T) {
+				tableNameDst, rp, encoder := setup(t, useKVProc)
+
+				runner.Exec(t, fmt.Sprintf("INSERT INTO %s VALUES ($1, $2)", tableNameDst), row1...)
+
+				keyValue2 := encoder(timeOneDayBackward, row2...)
+				keyValue3 := encoder(timeOneDayBackward, row3...)
+				require.NoError(t, insertRow(rp, keyValue3, keyValue2.Value))
+				expectedRows := [][]string{
+					{"1", "row1"},
+				}
+				runner.CheckQueryResults(t, fmt.Sprintf("SELECT * from %s", tableNameDst), expectedRows)
+
```

Comment:
perhaps also add a line for when `insertRow` adds a key with an older timestamp and stale prev value too?